.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -359,29 +364,48 @@
             Condition="'$(_ResizetizerIsAndroidApp)' == 'True'"
             IntermediateOutputPath="$(_MauiIntermediateSplashScreen)"
             MauiSplashScreen="@(MauiSplashScreen)"
+            InputsFile="$(_MauiSplashInputsFile)"
         />
         <ItemGroup Condition="'$(_ResizetizerIsAndroidApp)' == 'True'">
-            <MauiImage Include="@(MauiSplashScreen)" />
+            <_MauiSplashAssets Include="$(_MauiIntermediateSplashScreen)**\*" />
             <LibraryResourceDirectories Condition="Exists('$(_MauiIntermediateSplashScreen)')" Include="$(_MauiIntermediateSplashScreen)">
                 <StampFile>$(_ResizetizerStampFile)</StampFile>
             </LibraryResourceDirectories>
-            <FileWrites Include="$(_MauiIntermediateSplashScreen)**\*" />
         </ItemGroup>
 
         <!-- iOS, but not Catalyst -->
         <GenerateSplashStoryboard
-            Condition="'$(_ResizetizerIsiOSApp)' == 'True' and '$(TargetPlatformIdentifier)' != 'maccatalyst'"
-            OutputFile="$(_MauiIntermediateStoryboard)"
+            Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'"
```

Comment:
Does this run for catalyst now, or is `$(_ResizetizerIsiOSSpecificApp)` iOS-only?

Reply:
This is for iOS only. `_ResizetizerIsiOSApp` is for ios and ios-like (tvos, maccat)